### PR TITLE
[ASM] EXPANDR-6820

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-6313

## Description
Per Tal Bahalool, updating field type of multiselect

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@capanw  / @BigEasyJ , please approve, this is somewhat urgent

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Incident Fields
+
+- Updated the **ASM - Protocol** fields to multiSelect type.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md:4]:
```suggestion
Updated the **ASM - Protocol** fields to multiSelect type.
```

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@melamedbn / @YuvHayun , this has been approved by Xpanse.  Please merge when possible.  We realize that changing field type isn't common but it is necessary for this case to not cause future issues.

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Protocol.json [hunk 2]
```diff
@@ -13,13 +13,13 @@
     "locked": false,
     "name": "ASM - Protocol",
     "neverSetAsRequired": false,
-    "openEnded": false,
+    "openEnded": true,
```
Comment (YuvHayun) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Protocol.json:16]: Please elaborate more about the changes:
What exactly was the issue?
How did it fix it?
did you make sure it solved the issue?

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Incident Fields
+
+Updated the **ASM - Protocol** fields to multiSelect type.
```
Comment (YuvHayun) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md:4]: Let's also add "Fixed an issue where..." for customers who already updated to the problematic version.

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hey @johnnywilkes, see my comments.
Also, don't we want to update the RN of the problematic version to inform that this version contains a bug as per @ShahafBenYakir suggestion?

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Protocol.json (hunk 2 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Protocol.json:16]: explained in https://panw-global.slack.com/archives/C0489NUGBRB/p1706181195830429?thread_ts=1706173202.412929&cid=C0489NUGBRB

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md (hunk 3 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md]: @YuvHayun , updated [update RN](https://github.com/demisto/content/pull/32415/commits/07d37e7de80176e564a6e95000cb76de2c925bbd)

**Review by YuvHayun — APPROVED** [review]

**Comment by YuvHayun** [comment]
@johnnywilkes thx for the fix!
merging to internal pr.

**Pull request merged**